### ISSUE TITLE
Unable to turn up "schema $refs must follow this format" warning to error

### ISSUE THREAD

**Issue by jamescooke** [IBM/openapi-validator#77]
Am running v0.12.2 installed inside a Node 12.4 Docker container.

I'm getting a very helpful warning which looks like this:

```
  Message :   schema $refs must follow this format: *#/components/schemas*
  Path    :   paths./packages/{p_id}/clone.post.responses.200.content.application/json.schema.$ref
  Line    :   245

statistics

  Total number of errors   : 0
  Total number of warnings : 1

  warnings
   1 (100%) : schema $refs must follow this format
```

This warning has been very helpful because it helped me debug why a build with our testing team was failing. In order to get this check done early before builds go to our testing team, I'd like to turn this warning up to error - but I can't find the config for that.

From the code, it looks like this is raised by the walker, but when I set the config of walker to all errors:

```json
     "walker": {
       "no_empty_descriptions": "error",
      "has_circular_references": "error",
      "$ref_siblings": "error",
      "duplicate_sibling_description": "error"
     }
```

... then a warning is still returned.

Is there a way to turn this up to error?

**Comment by dpopp07** [comment]
@jamescooke Thanks for this issue. Off the top of my head I would have thought that this check was configurable but looking at the code, I see that it is not.

We were actually just discussing making this check an error by default yesterday morning. Whatever we decide there, this should be a configurable check. I will look into addressing that this week

**Comment by jamescooke** [comment]
@dpopp07 Thanks for another quick fix :+1:  Can confirm is working for me now.
